recipe/meta.yaml

```diff
@@ -1,54 +1,53 @@
-{% set version = "1.4.0" %}
+{% set version = "2.5.6" %}
 
 package:
   name: conda-lock
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/c/conda-lock/conda_lock-{{ version }}.tar.gz
-  sha256: 55c115a852e63395db8a6a90cf2892aab0543959f2d0bafb83158b6863831b8a
+  sha256: 323db8b6438b46648c4ba4dab81547851ee8266dbe913680f7837c94cc5f1023
 
 build:
   number: 0
-  skip: True  # [py<36]
-  script: {{ PYTHON }} -m pip install . -vv --no-deps
+  skip: True  # [py<38]
```

Comment:
Also not a blocker, but it seems `s390x` can be skipped as it's not listed in the [platforms](https://github.com/conda/conda-lock/blob/v2.5.6/pyproject.toml#L156).

Reply:
We should provide s390x if we can. I don't this section applying to us.

Reply:
Umm, I don't think these platforms are necessarily what's supported. We shuld keep s390x.